versioned_docs/version-1.5.0/install-kubeslice/yaml/yaml-controller-install.mdx

```diff
@@ -0,0 +1,277 @@
+addon-http-application-routing-default-http-backend is running at https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443/api/v1/namespaces/kube-system/services/addon-http-application-routing-default-http-backend/proxy
+addon-http-application-routing-nginx-ingress is running at http://40.125.122.238:80 http://40.125.122.238:443
+healthmodel-replicaset-service is running at https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443/api/v1/namespaces/kube-system/services/healthmodel-replicaset-service/proxy
+CoreDNS is running at https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443/ap
+```
+
+From the above output, copy the URL for the **Kubernetes control plane** to
+add it as the cluster endpoint in the `controller.yaml` file.
+
+For example,
+`https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443`.
+
+### Controller Configuration Parameters
+The following tables describe the configuration parameters used to install the KubeSlice Controller.
+
+| Parameter                                                    | Parameter Type | Description                                                                  | Required  |
+| ------------------------------------------------------------ | -------------- | ---------------------------------------------------------------------------- | --------- |
+| Kubeslice                                                    | Object         | This contains the controller and the namespace configuration parameters.     | Mandatory |
+
+#### KubeSlice Parameters
+
+This parameter contains the configuration object used in the
+[KubeSlice Controller YAML file](#create-a-controller-yaml).
+
+| Parameter                                              | Parameter Type | Description                                                            | Required  |
+| ------------------------------------                   | -------------- | ---------------------------------------------------------------------- | --------- |
+| [namespaceConfig](#namespace-configuration-parameters) | Object         | The labels and annotations associated with the namespace.              | Mandatory |
+| [controller](#controller-parameters)                   | Object         | The cluster where the KubeSlice Controller is installed.               | Mandatory |
+
+
+#### Namespace Configuration Parameters
+
+This object contains the labels and annotations associated with the namespace used in the [KubeSlice Controller YAML file](#create-a-controller-yaml).
+
+
+| Parameter   | Parameter Type   | Description                | Required |
+| ---------   | --------------   | -----------                | ---------|
+| labels      | String           | The namespace labels.      | Mandatory |  
```

Comment:
These two parameters are optional and data type is dictionary. Got this description verified by Gourish. 

![image](https://github.com/user-attachments/assets/9ba9f164-2f37-4849-8578-2350043e5d08)

Reply:
done